include/swift/AST/DiagnosticsParse.def

```diff
@@ -1962,6 +1962,12 @@ ERROR(attr_requires_concurrency, none,
       "concurrency is enabled",
       (StringRef, bool))
 
+//------------------------------------------------------------------------------
+// MARK: Consistency diagnostics
+//------------------------------------------------------------------------------
+ERROR(new_parser_failure, none,
+      "new parser has failed consistency checking; please see errors above", ())
```

Comment:
Do we currently create toolchains somewhere that would cause this error to be shown in Xcode? Because if it’s shown in Xcode, `please see errors above` is no longer correct if you see the error inline in the source file instead of in the terminal output of the compile.

Reply:
A +Asserts tool chain could see this in Xcode. We'd need to plumb diagnostics from Swift through to the C++ engine to improve things. We'll do that for ASTGen